src/controllers/expenses.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  addExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../services/exprensesService');
+
+function getAllExpensesAction(req, res) {
+  const expenses = getAllExpenses();
+
+  res.send(expenses);
+};
+
+function addExpenseAction(req, res) {
+  const expense = req.body;
+  const listPlaces = ['title', 'amount', 'category', 'note'];
+  const keysExpense = Object.keys(expense);
+  const { userId } = req.body;
+
+  const missingFields = listPlaces
+    .filter(field => !keysExpense.includes(field));
+
+  if (missingFields.length
+    || !getAllExpenses().filter((el) => el.userId === userId).length) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  addExpense(expense);
+
+  res.sendStatus(201);
+};
+
+function getExpenseAction(req, res) {
+  const { expenseId } = req.params;
```

Comment:
```suggestion
  const { id } = req.params;
```

Reply:
there is no `expenseId`
![image](https://user-images.githubusercontent.com/98764641/236408179-3380a3c9-4690-4731-a395-2d345920cf06.png)

Reply:
fix it everywhere

Reply:
done